flows-advanced/serverless-aws-chalice/README.md

```diff
@@ -0,0 +1,31 @@
+# Serverless AWS Chalice
+
+This recipe demonstrates how to use Prefect in an AWS lambda function managed by [Chalice](https://aws.github.io/chalice/). Prefect flows can be executed directly in the AWS Lambda or on a remote agent via an API call back to Prefect Cloud.
+
+## Getting Started
+
+First install the chalice framework from pip,
+
+```sh
+pip install -U chalice
+```
+
+Next, fill in `.chalice/config.json` from `.chalice/config.json.tpl` either manually or using the command below assuming you have `PREFECT_API_KEY` and `PREFECT_API_URL` set in your environment,
+
+```sh
+cat .chalice/config.json.tpl | envsubst > .chalice/config.json
```

Comment:
if you're on MacOS and you'll need to run `brew install gettext` as envsubstr is not a built-in for it, apparently.

Reply:
Hmm... it's probably overkill to specify dependencies for this, so I'll just dial back the instructions to the manual option.

Reply:
all good, I'm just looking for issues 😄